lib/templates/deploy.batch.aspnet.5.template

```diff
@@ -16,16 +16,15 @@ IF "%DEPLOYMENT_TARGET:~-1%"=="\" (
     SET DEPLOYMENT_TARGET=%DEPLOYMENT_TARGET:~0,-1%
 )
 
-IF NOT DEFINED SCM_DNX_VERSION (
```

Comment:
I'm no longer using environment variables. These markers [`{ASPNET5_RUNTIME}`, `{ASPNET5_ARCHITECTURE}`, `{ASPNET5_VERSION}`](https://github.com/projectkudu/KuduScript/blob/aspnet5/lib/templates/deploy.batch.aspnet.5.template#L26) will get replaced by the right values during script generation, so when you run it from the command line it should work.
Though this doesn't allow the users to set these options in AppSettings, but they can do it in global.json, so I figured we don't need to support AppSettings here.